docs/grpc/docs.md

```diff
@@ -1431,6 +1431,7 @@ Note: 1kB = 1 vector of size 256. |
 | search_allow_exact | [bool](#bool) | optional |  |
 | search_max_oversampling | [float](#float) | optional |  |
 | upsert_max_batchsize | [uint64](#uint64) | optional |  |
+| max_collection_vector_size | [uint64](#uint64) | optional |  |
```

Comment:
I think this one should have clear unit in the name.

Also it is possible that our users prefer working with megabytes right away.

WDYT?

Reply:
I was reminded by someone that this is an internal field, manipulated by our custom UI most likely.
Sticking to bytes is good enough then :+1: